### PR TITLE
Update paths for scripts when using homebrew

### PR DESCRIPTION
**Pull request by hjir**
This PR implements the following changes:

_Insert list of changes here_
- Adds a homebrew powerlog generate method that puts in the correct paths

_Remove the ones that are not applicable:_
* [x] I have added the necessary changes to `generate.py`.
* [x] I have added the necessary changes to `Makefile`.
* [x] I have checked that the generated tools work on my OS.
* [x] I followed the code style conventions.

### CODE REVIEW

**Comment by sarajuhosova** [comment]
I think let's have this as a separate `powerlog/package-brew` package. Then you also have to add it to the `make` and `clean` separately and fix the `generate_fernivy` to take an optional `type` parameter (so I guess `type="package"` and then you'd just call it with `type="package-brew"`).

File: generate.py [hunk 1]
```diff
@@ -65,28 +66,47 @@ def generate_perf(conf):
...
     """
     shutil.copytree("powerlog/backup/", "powerlog/package/")
+    shutil.copytree("powerlog/backup/", "powerlog/package-brew/")
+    # installation from source
+    generate_fernivy("powerlog",
+                     lambda line:
+                     line.replace(
+                         "$TOOL\"/backup/\"$TOOL\"_run.sh\"",
+                         "./powerlog/package/powerlog_run.sh")
+                     .replace("parser.py", "./powerlog/package/parser.py")
```
Comment (hjir) [inline generate.py:93]: hmm now that I think about it, I'm not sure if we wanna do this. when people install from source, do we want them to cd to `powerlog/package` and call it from there? or do we wanna have them call `powerlog/package/fernivy` from root?

Comment (sarajuhosova) [reply generate.py:93]: We actually want to be able to zip the `package` separately, so it should be relative to that.

**Comment by sarajuhosova** [comment]
It's legitimate!

File: generate.py (hunk 1 above)
Comment (hjir) [reply generate.py]: done:)

**Review by sarajuhosova — APPROVED** [review]

**Pull request merged**